Functions/Administration/xASL_adm_GetDeprecatedFields.m

```diff
@@ -97,6 +97,7 @@
     'bRunLongReg'                   'modules'   'bRunLongReg'                        '';...
     'ForceInclusionList'            'dataset'   'ForceInclusionList'                 '';...
     'bAutomaticallyDetectFSL'       'external'  'bAutomaticallyDetectFSL'            '';...
+	'ExternalQuantificationType'    'external'  'ExternalQuantificationType'         '';...
```

Comment:
ExternalQuantificationType should default to FSL?

But: don't we already have this parameter, and it defaults to ExploreASL quantification?
I propose:

Have a single parameter "QuantificationType" which can be ExploreASL/FSL/VABY.

(alternatively, we can have a boolean bExternalQuantification, but that is probably more work to keep track of)

Reply:
We have bUseBasilQuantification from previously, and we had an internal parameter to distinguish between BASIL and FABBER as this could be decided automatically based on multi-TE data.

I now added ExternalQuantificationType because FABBER/VABY cannot be decided based on data.

We of course want a single parameter to make things easier, but I didn't want ot make the decision without you.

I propose bUseBasilQuantification -> bExternalQuantification. And then keep ExternalQuantificationType as is.

We can of course just have bQuantificationType as ExploreASL/FSL/VABY/BASIL/FABBER - I have no preference and can change that to make it backwards compatible...

Reply:
A summary:
- [ ] bUseBasilQuantification is renamed to bExternalQuantification
- [ ] bUseBasilQuantification as an outdated variable gets renamed to bExternalQuantification
- [ ] ExternalQuantificationType is "BASIL" by default, but can be "FABBER" or "VABY" - we add a default and complain if another value is used